src/use_case/vote_out/VoteOutInteractor.java

```diff
@@ -1,4 +1,39 @@
 package use_case.vote_out;
 
+import entity.Game;
+
 public class VoteOutInteractor implements VoteOutInputBoundary {
+    final VoteOutDataAccessInterface userDataAccessObject;
+    final VoteOutOutputBoundary userPresenter;
+    final Game game;
+
+    public VoteOutInteractor(VoteOutDataAccessInterface userDataAccessObject, VoteOutOutputBoundary userPresenter, Game game) {
```

Comment:
From CA, the use case interactor should be calling the data access interface, which then creates the entity game (since we agreed a Game instance shouldn't be passed around the entire program, and we should instead be using a data access object (database) instead. 

So, instead, use the userDataAccessObject to get the game information and construct a game entity (instead of passing game as a parameter).